File: rest/addresses/list-post-example-1/list-post-example-1.4.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Services_Twilio($sid, $token);

// Create an address resource
$address = $client->account->addresses->create("Billing - Customer 123", array(
	"CustomerName" => "Customer 123",
	"Street" => "1 Hasselhoff Lane",
	"City" => "Berlin",
	"Region" => "Berlin",
	"PostalCode" => "10875",
	"IsoCountry" => "DE"
));
echo $address->sid;

File: rest/addresses/list-post-example-1/list-post-example-1.4.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Create a new address
@address = @client.account.addresses.create(
  friendly_name: 'Billing - Customer 123',
  customer_name: 'Customer 123',
  street: '1 Hasselhoff Lane',
  city: 'Berlin',
  region: 'Berlin',
  postal_code: '10875',
  iso_country: 'DE'
)
puts @address.sid
